src/include/libpbs.h

```diff
@@ -290,19 +301,19 @@ struct batch_reply {
 #define PBS_BATCH_CopyHookFile	85
 #define PBS_BATCH_DelHookFile	86
 #define PBS_BATCH_MomRestart	87
-#define PBS_BATCH_AuthExternal	88
+/* Unused -- #define PBS_BATCH_AuthExternal	88 */
 #define PBS_BATCH_HookPeriodic  89
 #define PBS_BATCH_RelnodesJob	90
 #define PBS_BATCH_ModifyResv	91
 #define PBS_BATCH_ResvOccurEnd	92
 #define PBS_BATCH_PreemptJobs	93
-#define PBS_BATCH_Cred          94
+#define PBS_BATCH_Cred		94
+#define PBS_BATCH_Authenticate	95
```

Comment:
You mentioned the merge of PBS_BATCH_AuthenResvPort and PBS_BATCH_AuthExternal in the design doc. Since the PBS_BATCH_AuthenResvPort will be unused, I believe it should be explicitly documented that the backward compatibility with all older pbs clients is broken. Otherwise, admins could be surprised.

Reply:
sure will update the design page with details.